atef/check.py

```diff
@@ -494,31 +544,35 @@ def _compare(self, value: Number) -> bool:
         return True
 
 
-ItemToChecks = Dict[
-    str,
-    Union[Comparison, Sequence[Comparison]],
-]
+@dataclass
+class IdentifierAndComparison:
```

Comment:
Do these need a `name` field so we can identify them against the other checklist items in `Configuration`

Reply:
Added in [4468a7c](https://github.com/pcdshub/atef/pull/27/commits/4468a7c979b5cceff488845ca45af05003d25317)